resources/views/partials/use-open-page-triple-cards.blade.php

```diff
@@ -0,0 +1,60 @@
+<p>Open textbooks in the B.C. Open Textbook Collection come in multiple formats. Many can be read online or delivered via PDF, EPUB (for most e-readers) and MOBI (for Kindle) format for free, or can be printed via our print-on-demand service for a nominal fee.</p>
+
+<section class="homepage-cards d-flex flex-row flex-wrap full-width">
+        <article class="homepage-cards-item col-md-4 mb-2" itemscope itemtype="http://schema.org/Article">
+            <a href="{{ get_permalink($get_use_open_triple_left_id) }}" class="img-link">
+                <div class="featured-topic row-fluid d-flex" style="background-image: url({{\App\Controllers\App::getThumbUrl($get_use_open_triple_left_id)}});">
+                    <h4 itemprop="name" class="blue-bkgd-special text-inverse col-sm mt-auto text-center m-0 p-2">{{get_the_title($get_use_open_triple_left_id)}}
```

Comment:
Please use `{!! !!}` so that titles with ampersands don't end up displaying as `&#038;`